tests/torch/test_sota_checkpoints.py

```diff
@@ -409,14 +410,15 @@ def test_eval(self, sota_checkpoints_dir, sota_data_dir, eval_test_struct: EvalR
                                                                           fp32_metric,
                                                                           diff_fp32,
                                                                           metric_type_from_json)
-        retval = self.threshold_check(is_ok,
+        retval = self.threshold_check(test,
```

Comment:
Rename `test` to `test_type`, please

Reply:
Removed. Since we are comparing the result with FP32 only, we don't need to calculate additional thresholds, so, we don't need to call threshold_check function.